generate/generate_file_check_test.go

```diff
@@ -204,7 +210,10 @@ func TestCheckOutdatedDetectsEmptyGenerateFileContent(t *testing.T) {
 		).String(),
 	)
 
+	s.ReloadConfig()
 	assertOutdatedFiles([]string{"test.txt"})
+
+	s.ReloadConfig()
```

Comment:
hmm if generate has the same issues with outdatedconfig maybe we could always reload the config also inside generate, or else I imagine this will be an issue most of the time we change files and usually we want s.Generate to be updated.

Reply:
I know it can make things slower, but not sure if in this case it will be worth the complexity of always ensuring the reload everywhere + when you forget I imagine that debugging can be challenging if you don't remeber this detail etc. But feel free to disregard, not a huge issue.